docs/self-hosted-appcircle/install-server/cloud-providers/azure.md

```diff
@@ -0,0 +1,197 @@
+
+#### Microsoft Azure Account
+
+You should have an active Azure account with appropriate permissions to launch virtual machines and work with other related services.
+
+#### Understanding of Azure Services
+
+A basic understanding of Azure services, particularly Virtual Machines, is beneficial.
+
+You should be familiar with virtual machine creation, networking, security groups, and storage configurations.
+
+##### 1. Networking and Security Configuration
+
+You might need to configure networking aspects such as Virtual networks, subnets, route tables, and security groups to properly integrate the instance within the network environment and manage access controls.
+
+##### 2. SSH Key Pairs for Secure Access
+
+You need an SSH key pair to access to the server that you will create securely.
+
+##### 3. Linux System Configuration
+
+Basic familiarity with Linux system configurations and commands is essential since this document will use Linux commands.
+
+## Creating an Appcircle Server from the server image
+
+After you meet all the requirements discussed above, you can follow the steps below to create an Appcircle server from the server image.
+
+- Log in to the Microsoft Azure with your account.
+
+- Head to the Virtual machines menu to create a virtual machine.
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/BE-4345-1-virtual-machine-menu.png' />
+
+- Click on the "+ Create" button and "Azure virtual machine" from the virtual machines dashboard.
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/BE-4345-2-create-vm-button.png' />
+
+You should fill out the required fields as per your needs. Please follow the below steps for a sample instance configuration.
+
+- Select the Subscription and Resource group for your needs.
+
+- Enter an virtual machine name in the "Instance details" field. For example, "My-Appcircle-Server".
+
+- Choose which region you want the server to be in.
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/BE-4345-3-create-vm-1.png' />
+
+- In order to select the Appcircle server image, click on the "See all images" button and search for the Appcircle server image.
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/BE-4345-4-create-vm-2-see-all-images.png' />
+
+- Search for "Appcircle" in the "Marketplace" tab and click on the "Select" button for the server image and select "Gen2".
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/BE-4345-5-create-vm-3-select-appcircle-image.png' />
+
+- We will use the `Standart_B4ms` size for our sample configuration since it meets the minimum requirements for the vCPU count and memory size.
+
+:::info
+For the details about minimum hardware requirements, you should see the [Hardware Requirements](/self-hosted-appcircle/install-server/docker#hardware-requirements) section.
+:::
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/BE-4345-11-machine-type.png' />
```

Comment:
> I think checking the information before the screenshot is better. @csonuryilmaz might decide that.

I agree with you here. Because the "minimum hardware requirements" info box is related to both the sentence above (machine type) and the screenshot below.

So it can be put in the middle of them. ✅